docs/understanding-airbyte/high-level-view.md

```diff
@@ -4,14 +4,15 @@ description: A high level view of Airbyte's components.
 
 # High-level View
 
-![3.048-Kilometer view](../.gitbook/assets/10-000-feet-view.png)
+![3.048-Kilometer view](../.gitbook/assets/understanding_airbyte_high_level_architecture.png)
 
-* `UI`: Acts as the control center for Airbyte. From the UI, you can configure new integration connections. You can also track the different syncing jobs and view logs.
+* `UI`: Airbyte's main control plane. From the UI, you can configure new integration connections. You can also track the different syncing jobs and view logs.
```

Comment:
I would say the API is the main control plane, not the UI. The UI is just a window into it.

Reply:
Ok, fair enough - control center and control plane feel the same to me. How does `Airbyte's main entrypoint to the control plane.` sound? Otherwise I can just leave the wording as it was before.

Reply:
FWIW I think our thinking has evolved since we wrote the initial wording. We initially thought the UI was the main entrypoint but since then we've seen lots of compelling API-first use cases. I would phrase it something like: 

```
`UI`: An easy-to-use graphical interface for interacting with the Airbyte API.  
`API`: Airbyte's main control plane. All operations in Airbyte such as creating sources, destinations, connections, managing configurations, etc.. are configured and invoked from the API. 
```

Reply:
Ok dope, I'll add this wording in.